app/controllers/api/v1/application_controller.rb

```diff
@@ -4,6 +4,14 @@ module API
   module V1
     class ApplicationController < ActionController::API
       include API::V1::DoorkeeperAuthentication
+      rescue_from ActiveRecord::RecordNotFound, with: :show_not_found_error
+      rescue_from Pagy::OverflowError, with: :show_not_found_error
```

Comment:
Not sure, if we can pass exceptions in array
```suggestion
      rescue_from [ActiveRecord::RecordNotFound, Pagy::OverflowError,], with: :show_not_found_error
```

Reply:
Nice catch, updated in [`fad7c96` (#56)](https://github.com/suho/google-search-ruby/pull/56/commits/fad7c96ef1dc335386d474c5abdcbad63a09b98d). 
More detail, we don't need `[` and `]`.
Just: 
```suggestion
      rescue_from ActiveRecord::RecordNotFound, Pagy::OverflowError, with: :show_not_found_error
```